plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -39,16 +39,35 @@ class CuraEngineBackend(Backend):
     #
     #   This registers all the signal listeners and prepares for communication
     #   with the back-end in general.
+    
+    executableName = "CuraEngine"
```

Comment:
Good question. I just wanted to use it here, because "CuraEngine" was mentioned multiple times. So I decided to define it once at one place.

Do you think it is needed to make it private here?